src/entities/trades/uniswap-v2/pairs/pairs.ts

```diff
@@ -118,13 +72,20 @@ export async function getAllCommonPairs({
     pairAddressList.map((target) => ({ target, callData }))
   )) as Multicall2TryAggregateResult[]
 
+  // Get the fees
+  const pairSwapFeeList = await getUniswapV2PairSwapFee({
```

Comment:
It might be more efficient to do just one call fetching both the reserves and swap fees from the `pairAddressList`.

Reply:
Take a look here :) [c0e6fd7](https://github.com/levelkdev/swapr-sdk/pull/91/commits/c0e6fd7942e915e74bbf9fd1970343bfe16f6828)